guides/v2.1/security/two-factor-authentication.md

```diff
@@ -0,0 +1,108 @@
+---
+group: security
+title: Two-Factor Authentication
+functional_areas:
+  - Configuration
+---
+
+The Magento Admin provides all access to your store, orders, and customer data. To further increase security to your Magento instance, add Magento Two-Factor Authentication (2FA), v3.0.0. Installing and enabling this module adds two-step authentication for all users attempting to access the Admin for all devices. All features and requirements are restricted to Admin user accounts, not extended to customer accounts.
```

Comment:
all access?      "full access to administrative users to manage all functionality for your store, orders, and customer data.  To improve security, you can add Magento 2FA, v3.0.0 to require two-step authentication for all Admin users accessing the Magento Admin.   When 2FA is enabled, it secures access to the Admin page for all administrative users on all devices.  The 2FA features are available only for Admin user accounts and do not extend to customer acocunts.

To further increase....

Consider -- You can add Magento 2FA to increase security for accessing this page by

Reply:
Rewriting intro